NBitcoin.TestFramework/NodeBuilder.cs

```diff
@@ -573,7 +573,7 @@ public void Broadcast(params Transaction[] transactions)
 			{
 				batch.SendRawTransactionAsync(tx);
 			}
-			rpc.SendBatch();
+			batch.SendBatch();
```

Comment:
wait how it could even work before?

Reply:
It didn't work before, in fact it threw an exception saying "use PrepareBatch".